src/backend/tests/teams.test.ts

```diff
@@ -46,7 +47,14 @@ describe('Teams', () => {
     expect(prisma.team.findUnique).toHaveBeenCalledWith({ where: { teamId }, ...teamQueryArgs });
   });
 
-  describe('editTeam', () => {
+  describe('setTeamMembers', () => {
+    test('setTeamMembers members not found', async () => {
+      jest.spyOn(prisma.team, 'findUnique').mockResolvedValue(prismaTeam1);
+
+      const res = await TeamsService.setTeamMembers(flash, sharedTeam1.teamId, [122, 55]);
+      expect(res).toThrow(new HttpException(404, 'User(s) with the following ids not found: 122, 55'));
```

Comment:
For the getUsers function, I tried to test the case when the given userIds do not exist, but I wasn't sure how to test on expecting HttpException to be thrown. (This line test fails) I would appreciate any feedback!